crates/egui/src/memory.rs

```diff
@@ -151,6 +155,32 @@ impl FocusDirection {
     }
 }
 
+#[derive(Clone, Debug)]
+#[cfg_attr(feature = "persistence", derive(serde::Deserialize, serde::Serialize))]
+#[cfg_attr(feature = "persistence", serde(default))]
+pub struct LayerTransform {
+    pub translation: Vec2,
+    pub scale: f32,
```

Comment:
The struct and fields needs docstrings; especially what order scale and translation are applied in